configure

```diff
@@ -9124,13 +9126,14 @@ $as_echo "#define USE_PAX_STORAGE 1" >>confdefs.h
   esac
 
 else
-  enable_pax=yes
-  
-  $as_echo "#define USE_PAX_STORAGE 1" >>confdefs.h
```

Comment:
not use `USE_PAX_STORAGE` anymore ?

Reply:
> not use `USE_PAX_STORAGE` anymore ?

When using the option `--enable-pax`, still add "#define USE_PAX_STORAGE 1" to `confdefs.h`. Just for when without the option, nothing adds.